WalletWasabi/Mono/OptionSet.cs

```diff
@@ -711,7 +711,7 @@ private bool ParseBundledValue(string f, string n, OptionContext c)
 							return true;
 						}
 					default:
-						throw new InvalidOperationException("Unknown OptionValueType: " + p.OptionValueType);
+						throw new InvalidOperationException($"Unknown {nameof(OptionValueType)}: " + p.OptionValueType);
```

Comment:
Don't need to concatenate. Is this work?
`$"Unknown {nameof(OptionValueType)}: {p.OptionValueType}"`